MapboxDirections/MBWaypoint.swift

```diff
@@ -9,6 +9,16 @@ open class Waypoint: NSObject, NSCopying, NSSecureCoding {
         return true
     }
     
+    /// :nodoc:
```

Comment:
Why aren’t these initializers documented? Because they’re only available for Objective-C? I think we have some examples of that elsewhere in MapboxDirections.swift or MapboxStatic.swift, where we still document the Objective-C-only methods but say they’re Objective-C-only.